File: Beecrowd/Beginner/2003 - 2176/2161 - Square Root of 10/Python/2161_SquareRootof10.py
denominatorRepetitions = float(input())

result = 0

for i in range(int(denominatorRepetitions)):
    if i == 0:
        result = 1 / (6 + result)
    else:
        result = 1 / (6 + result)

print(f"{result + 3:.10f}")

File: Beecrowd/Beginner/2003 - 2176/2161 - Square Root of 10/C/2161_SquareRootof10.c
#include <stdio.h>

int main() {
    double denominatorRepetitions;
    scanf("%lf", &denominatorRepetitions);

    double result = 0;

    for (int i = 0; i < (int)denominatorRepetitions; i++) {
        if (i == 0) {
            result = 1 / (6 + result);
        } else result = 1 / (6 + result);
    }

    printf("%.10lf\n", result + 3);

    return 0;
}
